### PR TITLE
set up server & mlab connection & create-react-app client

### PR DESCRIPTION
**Pull request by iqra-javed**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] tested on local host

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [X] My changes generate no new warnings

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const app = express();
+const port = process.env.PORT || 8080;
+require("dotenv").config();
+mongoose.connect(
+  `mongodb://${process.env.USERNAME}:${
+    process.env.PASSWORD
```
Comment (rusticlion) [inline server/server.js:8]: Appropriate values for local envs are documented in the project Slack.

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "url": "git+https://github.com/Lambda-School-Labs/CS10-workout-tracker.git"
+  },
+  "author": "Xang Thao, Leon Bates, Amanda Phillips, Iqra Javed",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Lambda-School-Labs/CS10-workout-tracker/issues"
+  },
+  "homepage": "https://github.com/Lambda-School-Labs/CS10-workout-tracker#readme",
+  "dependencies": {
+    "body-parser": "^1.18.3",
```
Comment (rusticlion) [inline server/package.json:21]: `body-parser` dependency is likely stale with new Express features.

**Review by rusticlion — APPROVED** [review]

**Review by emukupa — APPROVED** [review]
great job on getting and work together, for now! Keep it up

**Comment by emukupa** [comment]
Initial merge

**Pull request merged**